db/migrate/20170110090935_add_ems_to_miq_alert_status.rb

```diff
@@ -0,0 +1,19 @@
+class AddEmsToMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class MiqAlertStatus < ActiveRecord::Base; end
+
+  def up
+    add_column :miq_alert_statuses, :ems_id, :bigint
+    say_with_time("add ems_id to miq alert statuses") do
+      [Vm, VmOrTemplate, MiqTemplate, Host, EmsCluster, ContainerImage].each do |alert_resource|
```

Comment:
You don't want to reference an `app/model` class from migrations. Before, the `constantize` you were using was doing the same. (If @Fryguy has a different opinion, then I defer to his expertise)

also, I'm pretty sure `VmOrTemplate` will handle the `Vm` and `MiqTemplate` case. So those should not be needed.